pkg/piecestore/rpc/server/server_test.go

```diff
@@ -535,24 +538,22 @@ func NewTestServer(t *testing.T) *TestServer {
 		}
 	}
 
-	caS, err := provider.NewTestCA(context.Background())
-	check(err)
-	fiS, err := caS.NewIdentity()
-	check(err)
-	so, err := fiS.ServerOption()
+	serverID, err := node.NewFullIdentity(context.Background(), 12, 4)
 	check(err)
 
-	caC, err := provider.NewTestCA(context.Background())
+	so, err := serverID.ServerOption(serverID.ID.String())
 	check(err)
-	fiC, err := caC.NewIdentity()
+
+	nodeID, err := node.NewFullIdentity(context.Background(), 12, 4)
```

Comment:
--nit

`NewFullIdentity(...)`  returns a `FullIdentity`, I think the name `nodeID` is misleading; we've used `fi` in other places but otherwise, I'd use something with the word "identity" (e.g. `fullIdentity`) rather than the abbreviation "ID".